### PR TITLE
Getting started

### PR DESCRIPTION
**Pull request by t0mhaines**
Currently takes you up to an initial connection

### CODE REVIEW

File: getting_started.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+----
+
+### What?
+This is a getting started guide for the Dwyl smart home system.
+It contains everything you need to set up your own system;
+either for development or production.
+
+It contains no infomation on any of the protocols used or how the system works,
+this can be found here: ***TODO: Design documentation***
```
Comment (nelsonic) [inline getting_started.md:22]: https://github.com/dwyl/smart-home-security-system/pull/8 👍

File: getting_started.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+| Hardware | Notes |
+|----------|--------|
+Raspberry Pi 3+ / Zero | Or anything better with WiFi support
+PN532 Development board | Find on with UART support
+Necessary Connectors | You'll need at least 4 female-female wires <sup>1</sup>
+Hub Server | Anything that can run a server, another Raspberry Pi, or I used my laptop for development.
+
+1 : You can also use a breadboard with normal F-to-M jumper wires
+https://imgur.com/a/OQK8zMO
```
Comment (nelsonic) [inline getting_started.md]: Your wiring photo looks good. Maybe just add it inline so it's immediately visible. e.g:
```suggestion
![NFC-Reader-Wired-using-Breadboard](https://i.imgur.com/g7JqsfO.png)
```
NFC-Reader-Wired-using-Breadboard

File: getting_started.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+You'll need Elixir and Nerves setup on your computer. 
+If you don't have this, please follow the first part of our 
+[Learn Nerves](https://github.com/dwyl/learn-nerves) tutorial.
+
+You'll also need postgres running where your hosting the hub server.
+
+API Development software. Our hub currently does not have a GUI,
+only a REST API. 
+I recommend using [Insomnia Core](https://insomnia.rest/) as there's a 
+workspace already setup in the Hub repo.
```
Comment (nelsonic) [inline getting_started.md:74]: Could be worth spending an hour summarising your knowledge and reasoning for using insomnia.

File: getting_started.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+1. Clone the necessary repositories:
+
+        git clone https://github.com/dwyl/smart-home-auth-server
+        git clone https://github.com/dwyl/smart-home-firmware
+
+2. If you haven't yet, set your DWYL_API_KEY/AUTH_API_KEY
+
+        export AUTH_API_KEY=<Your Key> 
+
+3. Wire up the Raspberry Pi to the PN532 board over UART
```
Comment (nelsonic) [inline getting_started.md]:
```suggestion
3. Wire up the Raspberry Pi to the PN532 board over Universal Asynchronous Receiver/Transmitter (UART) protocol
```

File: getting_started.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+
+    mix setup
+    mix phx.server
+
+This should be all you have to do.
+
+
+### Setup REST API developent
+
+**This is a pain, we need to fix this**
```
Comment (nelsonic) [inline getting_started.md:119]: agreed.  if you don't mind, please create an issue to fix. 👍

Comment (t0mhaines) [reply getting_started.md:119]: #11

Comment (nelsonic) [reply getting_started.md:119]: Thanks for creating. 🙌

**Review by nelsonic — APPROVED** [review]
@th0mas getting started looking good. Nice work! Thanks! 🥇

**Pull request merged**